packages/mobile/src/analytics/ValoraAnalytics.test.ts

```diff
@@ -83,11 +83,11 @@ describe('ValoraAnalytics', () => {
       sHasVerifiedNumber: false,
       sLanguage: 'es-419',
       sLocalCurrencyCode: 'PHP',
-      sOtherTenTokens: 'TT:0,MOO:0',
```

Comment:
@jeanregisser This change caused 0-balance tokens to not show up here anymore. Is that ok? Right now in production we are sending the same tokens for everyone with zero balance, right?

Reply:
Yes I think this is fine.